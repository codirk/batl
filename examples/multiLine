First line ${echo -n ok}
Second line ${echo -n ok}
