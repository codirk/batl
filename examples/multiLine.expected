First line ok
Second line ok
